test/hotspot/jtreg/compiler/vectorization/TestRoundVectFloat.java

```diff
@@ -49,7 +49,13 @@ public static void main(String args[]) {
   }
 
   @Test
-  @IR(applyIf = {"UseAVX", " > 1"}, counts = {IRNode.ROUND_VF , " > 0 "})
+  @IR(applyIfPlatform = {"x64", "true"},
+      applyIf = {"UseAVX", " > 1"},
+      counts = {IRNode.ROUND_VF , " > 0 "})
+  @IR(applyIfPlatform = {"riscv64", "true"},
+      applyIfCPUFeature = {"rvv", "true"},
```

Comment:
This `applyIfCPUFeature = {"rvv", "true"},` seems unnecessary? You have already requires `(os.simpleArch == "riscv64" & vm.cpu.features ~= ".*rvv.*")`.

Reply:
Yes, it's unnecessary.
But we require `MaxVectorSize >= 32` below, so maybe it's better to keep it for readability.

Reply:
I guess it's not a big issue. I just find it's not the case when looking at changes made in file : test/hotspot/jtreg/compiler/vectorization/TestRoundVectRiscv64.java